### PR TITLE
[RW-6313] Install additional AoU genomics tools

### PR DESCRIPTION
**Pull request by calbach**
- Install gatk
- Install bolt-lmm
- Install regenie
- Install Nirvana (requires dot net)
- Install R packages: SAIGE, GENESIS
- Update Plink version

### CODE REVIEW

**Comment by calbach** [comment]
@ericsong @als364 ptal

File: terra-jupyter-aou/Dockerfile [hunk 1]
```diff
@@ -77,13 +88,64 @@ RUN mkdir -p /tmp/plink && \
   cd $HOME && \
   rm -rf /tmp/plink
 
+# GATK install
+ENV GATK_VERSION 4.1.4.1
```
Comment (Qi77Qi) [inline terra-jupyter-aou/Dockerfile:92]: would it make sense to just base the aou image off of gatk one?

File: terra-jupyter-aou/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+## 1.1.2 - 2021-04-21
+
+- Install gatk
+- Install bolt-lmm
+- Install regenie
+- Install Nirvana
+- Install R packages: SAIGE, GENESIS
+- Update Plink version
+
+Image URL: `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.1`
```
Comment (ericsong) [inline terra-jupyter-aou/CHANGELOG.md:10]: should this be 1.1.2?

**Review by als364 — APPROVED** [review]
LGTM.

**Review by ericsong — APPROVED** [review]
LGTM

File: terra-jupyter-aou/Dockerfile (hunk 1 above)
Comment (rtitle) [reply terra-jupyter-aou/Dockerfile:92]: Just a note GATK 4.2.0.0 is out, we have a ticket to update `terra-jupyter-gatk` as well:

https://github.com/broadinstitute/gatk/releases

Comment (calbach) [reply terra-jupyter-aou/Dockerfile]: Oh wow, I didn't notice that the GATK image already included R and Python (I guess our image was effectively forked from there, a while back). I switched to gatk as the base and greatly the AoU Dockerfile, great suggestion!

I think we're fine to just accept the upgrade whenever it happens, the base version is fine for now (I don't think anyone is clamoring for anything specific from gatk for the AoU genomics demo project phase).

File: terra-jupyter-aou/CHANGELOG.md (hunk 2 above)
Comment (calbach) [reply terra-jupyter-aou/CHANGELOG.md]: Thanks - yes.

**Comment by calbach** [comment]
@Qi77Qi or @rtitle I'll await your approval on this (I'm doing some additional local testing in the meantime)

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**